vumi/transports/vumi_bridge/tests/test_vumi_bridge.py

```diff
@@ -65,7 +64,7 @@ def get_next_request(self):
 
 class TestGoConversationTransport(TestGoConversationTransportBase):
 
-    transport_class = GoConversationClientTransport
+    transport_class = GoConversationTransport
```

Comment:
We need to delete these old client transport tests. They're not failing currently because they're accidentally not being run (because the both this class and the server test class are now called `TestGoConversationTransport`).